apps/builder/app/shared/instance-utils.ts

```diff
@@ -1621,3 +1589,40 @@ export const findClosestSlot = (
     }
   }
 };
+
+export type Insertable = {
+  parentSelector: InstanceSelector;
+  position: number | "end";
+};
+
+export const findClosestInsertable = (
```

Comment:
what's the difference between findClosestInsertable and findClosestDroppableTarget?

Reply:
findClosestDroppableTarget is previous implementation which did not access stores. Here the utility access stores for simplicity of usage. findClosestDroppableTarget is used only in this modules, not exported anymore. Will probably refactor further later.

Reply:
oh, you probably need to inline that into that function, because its confusing from naming perspective